charts/hawkbit/values.yaml

```diff
@@ -41,7 +41,7 @@ updateStrategy:
   type: Recreate
 
 nameOverride: ""
-fullnameOverride: ""
+fullnameOverride: "eclipse-hawkbit"
```

Comment:
Is this change needed? I think it's better to keep the old value and let the default be applied (including the release name).

And, could you do a rebase? There was an update of the chart version.